src/main/resources/root.jsp

```diff
@@ -26,10 +31,10 @@
             version: '<%= Jahia.VERSION %>',
             contextPath: '${contextPath}',
             user: {
-                username: '${functions:escapeJavaScript(currentUser.name)}',
-                fullname: '${functions:escapeJavaScript(user:fullName(currentUser))}',
-                firstName: '${functions:escapeJavaScript(currentUser.properties['j:firstName'])}',
-                lastName: '${functions:escapeJavaScript(currentUser.properties['j:lastName'])}',
+                username: currentUsername,
+                fullname: userFullname,
```

Comment:
The taglib user:fullname could not be fixed ?
That function make usage of escapeXml for security reason to avoid injecting code in the name